clang/test/dpct/helper_files_ref/include/rng_utils.hpp

```diff
@@ -174,6 +176,235 @@ template <typename engine_t> class rng_generator {
 };
 
 } // namespace device
+
+namespace host {
+enum class random_engine_type {
+  philox4x32x10,
+  mrg32k3a,
+  mt2203,
+  mt19937,
+  sobol
+};
+
+/// The random number generator on host.
+class rng_host {
+public:
+int create_generator(const random_engine_type &type) {
+  if (type == _type) {
+    return 0;
+  }
+  destroy_generator();
+  _type = type;
+  switch (type) {
+  case random_engine_type::philox4x32x10:
+    _philox4x32x10_engine = std::make_shared<oneapi::mkl::rng::philox4x32x10>(
+        dpct::cpu_device().default_queue(), _seed);
```

Comment:
Why using cpu_device here? I think we should use default queue.

Reply:
Fixed, and extract the logic to the initial value of `_queue`: `sycl::queue _queue{dpct::get_default_queue()};`